<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KNotifyConfigActionsWidgetBase</class>
 <widget class="QWidget" name="KNotifyConfigActionsWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>202</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="QToolButton" name="Sound_play">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="KUrlRequester" name="Sound_select">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Select the sound to play</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="Sound_check">
     <property name="text">
      <string>Play a &amp;sound</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QCheckBox" name="Popup_check">
     <property name="text">
      <string>Show a message in a &amp;popup</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>Sound_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>Sound_play</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Sound_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>Sound_select</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>18</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
